setup.py

```diff
@@ -211,6 +211,31 @@ def _run_domain_js(self, domain):
         'distutils.commands': [
             'build_sphinx = sphinx.setup_command:BuildDoc',
         ],
+        'sphinx.builders': [
+            'dummy = sphinx.builders.dummy:DummyBuilder',
+            'html = sphinx.builders.html:StandaloneHTMLBuilder',
+            'dirhtml = sphinx.builders.html:DirectoryHTMLBuilder',
+            'singlehtml = sphinx.builders.html:SingleFileHTMLBuilder',
+            'pickle = sphinx.builders.html:PickleHTMLBuilder',
+            'json = sphinx.builders.html:JSONHTMLBuilder',
+            'web = sphinx.builders.html:PickleHTMLBuilder',
+            'htmlhelp = sphinx.builders.htmlhelp:HTMLHelpBuilder',
+            'devhelp = sphinx.builders.devhelp:DevhelpBuilder',
+            'qthelp = sphinx.builders.qthelp:QtHelpBuilder',
+            'applehelp = sphinx.builders.applehelp:AppleHelpBuilder',
+            'epub = sphinx.builders.epub:EpubBuilder',
+            'epub3 = sphinx.builders.epub3:Epub3Builder',
```

Comment:
Entry points for the built-in builders were removed as requested by @shimizukawa. This could be added back at a later point.